src/qtiCommonRenderer/renderers/interactions/PortableCustomInteraction.js

```diff
@@ -179,18 +183,20 @@ var getState = function getState(interaction) {
     return _getPciRenderer(interaction).getState(interaction);
 };
 
+var getData = function getData(customInteraction, data) {
+    //remove ns + fix media file path
+    var markup = data.markup;
+    markup = util.removeMarkupNamespaces(markup);
+    markup = PortableElement.fixMarkupMediaSources(markup, this);
+    data.markup = markup;
```

Comment:
side effect

Reply:
I did not change it it was only prettified